DeepSpeech.py

```diff
@@ -1588,6 +1590,50 @@ def end(self, session):
         dropout_rates[5]: 0.,
     }
 
+
+    # Progress Bar
+    def update_progressbar(set_name):
+        if not hasattr(update_progressbar, 'current_set_name'):
+            update_progressbar.current_set_name = None            
+
+        if update_progressbar.current_set_name != set_name:
+
+            # finish prev pbar if it exists
+            if hasattr(update_progressbar, 'pbar') and update_progressbar.pbar:
+                update_progressbar.pbar.finish()
+
+            update_progressbar.total_jobs = None
+            update_progressbar.current_job_index = 0
+
+            current_epoch = COORD._epoch-1
+
+            if job.set_name == "train":
+                log_info('Training epoch %i...' % current_epoch)
+                update_progressbar.total_jobs = COORD._num_jobs_train
+                total_samples = model_feeder.train.total_batches*FLAGS.train_batch_size
+                log_info("Total samples: %i, total jobs: %i" % (total_samples, update_progressbar.total_jobs))
```

Comment:
The number of jobs is already listed on the progress bar and the number of total samples isn't terribly useful, I'd remove these lines to make the log more compact.